browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -173,6 +198,29 @@ void SpeedreaderTabHelper::HideBubble() {
   }
 }
 
+void SpeedreaderTabHelper::OnDistillComplete() {
+  // Perform a state transition
+  auto* entry = web_contents()->GetController().GetLastCommittedEntry();
+  DCHECK(entry);
```

Comment:
this is a discouraged pattern, see https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md#check_dcheck_and-notreached

Reply:
gotcha, fixed